js/collections/payment-requests.js

```diff
@@ -0,0 +1,26 @@
+var app = app || {};
+
+app.collections = app.collections || {};
+
+app.collections.PaymentRequests = (function() {
+
+    'use strict';
+
+	return Backbone.Collection.extend({
+
+		localStorageKey: 'payment-requests',
+
+		fetch: function() {
+
+			var data = localStorage.getItem(this.localStorageKey);
+
+			if (data) {
+				return this.set(JSON.parse(data));
+			}
+		}
+	});
+
+})();
+
+
+app.paymentRequests = new app.collections.PaymentRequests();
```

Comment:
This line should be moved to [init.js](https://github.com/Learn-by-doing/crypto-terminal/blob/4f82d9647382f5682abe85452dcdef50a229c616/js/init.js).